assignments/closure.js

```diff
@@ -1,20 +1,43 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
-
+function female() {
```

Comment:
Good job!  When you create a function, scope is created. Scope is where the JavaScript lives and is executed. The closure is a collection of all the variables in the scope that were created when the function was created.


When a function gets created and passed around or returned from another function, it carries a backpack with it. And in the backpack are all the variables that were in scope when the function was declared. Closure is the backpack that holds the variables for later.